src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerUdpClient.cs

```diff
@@ -50,28 +45,18 @@ public int Send(byte[] buffer, int offset, int count)
 
         /// <inheritdoc/>
         public void Dispose()
-        {
-            this.Dispose(true);
-            GC.SuppressFinalize(this);
-        }
-
-        /// <summary>
-        /// Releases the unmanaged resources used by this class and optionally releases the managed resources.
-        /// </summary>
-        /// <param name="disposing"><see langword="true"/> to release both managed and unmanaged resources; <see langword="false"/> to release only unmanaged resources.</param>
-        protected virtual void Dispose(bool disposing)
         {
             if (this.disposed)
             {
                 return;
             }
 
-            if (disposing)
-            {
-                this.client.Dispose();
-            }
+            this.client.Dispose();
 
             this.disposed = true;
         }
+
+        public override string ToString()
+            => $"{this.host}:{this.port}";
```

Comment:
Is this for debugging purpose? (it's a bit weird, maybe consider `JaegerUdpClient("udp://host:port")`?)

Reply:
I think it is just there for debugging. I'll remove it.